src/EventListener/FileMetaInformationListener.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/*
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2016 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\NewsBundle\EventListener;
+
+use Contao\CoreBundle\Framework\ContaoFrameworkInterface;
+use Contao\Database;
+use Contao\Database\Result;
+
+/**
+ * Provides file meta information for the request.
+ *
+ * @author Andreas Schempp <https://github.com/aschempp>
+ */
+class FileMetaInformationListener
+{
+    /**
+     * @var ContaoFrameworkInterface
+     */
+    private $framework;
+
+    /**
+     * Constructor.
+     *
+     * @param ContaoFrameworkInterface $framework
+     */
+    public function __construct(ContaoFrameworkInterface $framework)
+    {
+        $this->framework = $framework;
+    }
+
+    /**
+     * Returns the page record related to the given table and ID.
+     *
+     * @param string $table
+     * @param int    $id
+     *
+     * @return Result|false
+     */
+    public function onAddFileMetaInformationToRequest($table, $id)
+    {
+        if ('tl_news' === $table) {
```

Comment:
It's actually the second call that should be on `tl_news_archive`. Fixed in e596ce7b4bbfa9089993512435a419438a4d272e